packages/web/src/pages/sign-on/components/ProfileForm.tsx

```diff
@@ -163,15 +204,14 @@ const ProfileForm = (props) => {
           <Spring
             from={{ opacity: 0 }}
             to={{ opacity: 1 }}
-            leave={{ opacity: 0 }}
             config={{ duration: 200 }}
           >
             {(animProps) => (
               <div style={animProps} className={styles.suggestTwitter}>
                 <TwitterLogin
-                  onFailure={(...args) => console.log(args)}
-                  onSuccess={props.onTwitterLogin}
-                  className={styles.hideTwitterButton}
+                  onFailure={console.log}
+                  onSuccess={onTwitterLogin as any}
+                  // className={styles.hideTwitterButton}
```

Comment:
Commented out

Reply:
Ohhh yeah thanks for reminding me to check on this!